web/skins/classic/js/skin.js

```diff
@@ -300,6 +301,53 @@ if ( currentView != 'none' && currentView != 'login' ) {
       }
     });
 
+    // Manage visible object & control button (when pressing a button)
+    $j("[data-flip-сontrol-object]").click(function() {
+      let objIconButton = $j(this).find("i");
```

Comment:
I had this idea, but decided not to do it (at a slight cost to performance), because:
1. "$j(this)" doesn't appear that many times, only three times in one function and four times in another.
2. Now it’s more clear.
3. Add two lines of code.

But now I doubt that I did the right thing....
Maybe it's really worth changing the code?

Reply:
I added
const _this_ = $j(this);